nexus/external-api/src/lib.rs

```diff
@@ -3530,6 +3539,175 @@ pub trait NexusExternalApi {
         rqctx: RequestContext<Self::Context>,
         params: TypedBody<params::DeviceAccessTokenRequest>,
     ) -> Result<Response<Body>, HttpError>;
+
+    // Webhooks
+
+    /// List webhook event classes
+    #[endpoint {
+        method = GET,
+        path = "/v1/webhooks/event-classes",
+        tags = ["system/webhooks"],
+    }]
+    async fn webhook_event_class_list(
+        rqctx: RequestContext<Self::Context>,
+        pag_params: Query<
+            PaginationParams<EmptyScanParams, params::EventClassPage>,
+        >,
+        filter: Query<params::EventClassFilter>,
+    ) -> Result<HttpResponseOk<ResultsPage<views::EventClass>>, HttpError>;
+
+    /// List webhook receivers.
+    #[endpoint {
+        method = GET,
+        path = "/v1/webhooks/receivers",
+        tags = ["system/webhooks"],
+    }]
+    async fn webhook_receiver_list(
+        rqctx: RequestContext<Self::Context>,
+        query_params: Query<PaginatedByNameOrId>,
+    ) -> Result<HttpResponseOk<ResultsPage<views::WebhookReceiver>>, HttpError>;
+
+    /// Fetch webhook receiver
+    #[endpoint {
+        method = GET,
+        path = "/v1/webhooks/receivers/{receiver}",
+        tags = ["system/webhooks"],
+    }]
+    async fn webhook_receiver_view(
+        rqctx: RequestContext<Self::Context>,
+        path_params: Path<params::WebhookReceiverSelector>,
+    ) -> Result<HttpResponseOk<views::WebhookReceiver>, HttpError>;
+
+    /// Create webhook receiver.
```

Comment:
Remove periods at the end of these, please. They end up in the docs site sidebar. Wording otherwise looks good to me. Can't really get any shorter.

<img width="283" alt="image" src="https://github.com/user-attachments/assets/02282fc2-65d4-44b6-8e9d-3b9e8f0dff10" />

Reply:
Actually one wording change: remove "a" from "List delivery attempts to a webhook receiver"